package.nls.json

```diff
@@ -480,7 +480,9 @@
     "DataScience.interactiveWindowModeBannerSwitchAlways": "Always",
     "DataScience.interactiveWindowModeBannerSwitchNo": "No",
     "DataScience.ipykernelNotInstalled": "IPyKernel not installed into interpreter {0}",
-    "DataScience.needIpykernel6": "Ipykernel 6 is needed for debugging, click Install to continue. Or you can run 'pip install ipykernel==6.0.3/conda install ipykernel=6'",
+    "DataScience.needIpykernel6": "Ipykernel setup required for this feature",
```

Comment:
I'm sure this has been discussed, but could this message be more specific, and/or could we update the variable name `needIpykernel6` to match the new message?